### PR TITLE
Updated table loader to connect to secondary if required

### PR DESCRIPTION
**Pull request by whitew1994WW**

### CODE REVIEW

File: datadayessentials/authentications/_authentications.py [hunk 1]
```diff
@@ -108,17 +108,17 @@ def connect(self):
...
         database = database_info["database"]
-        
+        port = database_info['port'] if 'port' in database_info else None
+        application_intent = database_info['application_intent'] if 'application_intent' in database_info else None
+
+        connection_string = "DRIVER={ODBC Driver 17 for SQL Server};SERVER=" + server + ";DATABASE=" + database + ";ENCRYPT=yes;UID=" + self.credentials["USERNAME"] + ";PWD=" + self.credentials["PASSWORD"]
+        if port:
+            connection_string += f";PORT={port}"
+        if application_intent:
+            connection_string += f";ApplicationIntent={application_intent}"
```
Comment (Colin-Hanley) [inline datadayessentials/authentications/_authentications.py:118]: is there instances where port and application intent isnt available ? I feel as though if they are not there a error should be raised.

**Review by Colin-Hanley — CHANGES_REQUESTED** [review]
Question regarding port and application intent

File: datadayessentials/authentications/_authentications.py (hunk 1 above)
Comment (whitew1994WW) [reply datadayessentials/authentications/_authentications.py:117]: Both of these arent required for the connection to the readable secondary (what we are doing now)

They are optional extra paramters that can be passed to the database connection string. We could mandate them, but it seems excessive when it isnt needed.

**Review by Colin-Hanley — APPROVED** [review]
happy as larry

**Pull request merged**